ow_plexil/src/plans/README.md

```diff
@@ -60,3 +69,26 @@ The following represent the success criteria of the Reference Mission 2 plan:
 response to a fault
 	- Faults can be injected via rqt, a python script, or the command line
  ([Fault Injection Tutorial](https://github.com/nasa/ow_simulator/blob/master/ow_faults/README.md))
+
+
+### IdentifySampleLocationDemo ###
+
+The following represent the success criteria of the IdentifySampleLocationDemo plan: 
+- In the Rqt image viewer select the /sample_location topic. Once a point is selected
```

Comment:
I cannot find this selection (see screenshot).  Should we be using the melodic-devel branch of ow_simulator?
![rviz-topics](https://user-images.githubusercontent.com/9061371/129594606-ea649d83-eb6f-489b-9b1a-a91d779d2ff0.png)